roles/confluent.test/molecule/confluent-kafka-kerberos-customcerts-rhel/verify.yml

```diff
@@ -0,0 +1,108 @@
+---
+- name: Verify - kafka_broker
+  hosts: kafka_broker
+  gather_facts: false
+  tasks:
+    - import_role:
+        name: confluent.test
+        tasks_from: check_property.yml
+      vars:
+        file_path: /etc/kafka/server.properties
+        property: sasl.enabled.mechanisms
+        expected_value: GSSAPI
+
+    - import_role:
+        name: confluent.test
+        tasks_from: check_package.yml
+      vars: 
+        package_name: confluent-kafka-2.12-5.5.0-1.noarch
```

Comment:
this package name will need to be updated on each branch, is there a way to not include the "5.5.0-1"